gem/lib/pupperware/spec_helper.rb

```diff
@@ -230,6 +230,14 @@ def wait_on_puppetdb_status(seconds = 240)
     end
   end
 
+  ######################################################################
+  # PE Console Services Helper
+  ######################################################################
+  def unrevoke_console_admin_user(postgres_container_name="postgres")
+    postgres_command = "UPDATE subjects SET is_revoked = 'f' WHERE login='admin';"
+    docker_compose("exec -T #{postgres_container_name} psql --username=puppetdb --dbname=pe-rbac -c \"#{postgres_command}\"")
```

Comment:
Does this work properly on Windows?

Reply:
Also - note that we already have some Postgres helpers above that do things a little differently at https://github.com/puppetlabs/pupperware/pull/68/commits/d613d7be4f41552021a00bb1ee48c96f8de3a29d#diff-4cd2814855bdc73a2d390ef1cf973ae7R186

Reply:
Yes, this is method is pulled from https://github.com/puppetlabs/pe-console-services/pull/203, which is passing on Windows.

Reply:
Good call on the postgres helper, let me update to follow that style.